build/ToolInfo.props

```diff
@@ -0,0 +1,10 @@
+<Project>
```

Comment:
This file feels like it's a bit overloaded. NuGetPackagesDir should probably reside in dir.props directly, or in a NuGet-specific file. The remainder describes the Crossgen tool explicitly.